R/class-workbook.R

```diff
@@ -3780,7 +3780,6 @@ wbWorkbook <- R6::R6Class(
       if (length(cols) == 0L) {
         return(self)
       }
-      assert_class(hidden, "logical", arg_nm = "hidden")
```

Comment:
I guess we can relax this, but logical is correct. It can be 0/1 or true/false. Any value greater 0 is interpreted as 1, but provides no additional benefit.

Reply:
openxlsx2:::assert_class(c(0, 1), class = "logical")
#> Error x must be of class logical.

I think we can let it slide for now?